odpf/stencil/v1/stencil.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+package odpf.stencil.v1;
+
+import "google/api/annotations.proto";
+import "google/api/field_behavior.proto";
+import "google/protobuf/descriptor.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+
+option go_package = "github.com/odpf/stencil/server/api/v1/pb";
```

Comment:
whats the reason to have `/pb` at the end ?

Reply:
Lets change this to something like - `github.com/odpf/proton/stencil/v1` ?

Reply:
@harikrishnakanchi Yes, package name should be `github.com/odpf/proton/stencil/v1`. Will this require the server folder change is it?

Reply:
then we have to remove odpf folder and move stencil folder to root folder.

Reply:
as discussed we can have `github.com/odpf/proton/stencil/v1` package but don't need to change the folder structure for now, we will do that as part of another issue already in discussion at #23

Reply:
Has anyone tried using package alias using `;`? Because having version at the end of package path will generate the import paths as `v1/v2/...` which I don't think is very nice while importing generated structs. So something like
```
option go_package = "github.com/odpf/stencil/server/api/v1;servicepb";
```

Reply:
We can always import in golang like `stencil "github.com/odpf/proton/stencil/v1"` ?

Reply:
we are going with `github.com/odpf/proton/stencil/v1;stencilv1`.
Decided after looking at the buf docs for managed mode, which we might use in future - 
https://docs.buf.build/generate/managed-mode#:~:text=if%20the%20protobuf%20file's%20package%20declaration